tldr/cross_file_calls.py

```
@@ -3880,62 +4442,375 @@ def _build_php_call_graph(
                     if len(parts) == 2:
                         class_name, method = parts
-                        # Try to resolve class name via imports
                         if class_name in import_map:
                             namespace, resolved_class = import_map[class_name]
-                            # Look for Class::method in index
+                            # Look for Class::method in index (ambiguity-safe)
+                            _candidates = []
                             for key, file_path in func_index.items():
                                 if isinstance(key, tuple) and len(key) == 2:
                                     _, name = key
                                     if name == method or name == f"{resolved_class}::{method}":
```

Ruff (0.15.10)

[warning] Consider merging multiple comparisons: `name in (method, f"{resolved_class}::{method}")`. Use a `set` if the elements are hashable.

Merge multiple comparisons

(PLR1714)